src/dlgRoomExits.cpp

```diff
@@ -965,15 +959,25 @@ void dlgRoomExits::setIconAndToolTipsOnSpecialExit(QTreeWidgetItem* pSpecialExit
         // A valid exit roomID number:
         int exitAreaID = pExitToRoom->getArea();
         bool outOfAreaExit = (exitAreaID && exitAreaID != mAreaID);
+        bool exitRoomLocked = pExitToRoom->isLocked;
         QString exitAreaName;
         if (outOfAreaExit) {
             exitAreaName = mpHost->mpMap->mpRoomDB->getAreaNamesMap().value(exitAreaID);
         }
 
         // This is the toolTip text for the roomID number column (and the
-        // status icon)
-        const QString roomIdToolTipText{generateToolTip(pExitToRoom->name, exitAreaName, outOfAreaExit, pExitToRoom->getWeight())};
-        pSpecialExit->setIcon(ExitsTreeWidget::colIndex_exitStatus, showIconOnExitStatus ? (outOfAreaExit ? mIcon_otherAreaExit : mIcon_inAreaExit) : QIcon());
+        // status icons)
+        const QString roomIdToolTipText{generateToolTip(pExitToRoom->name, exitAreaName, exitRoomLocked, outOfAreaExit, pExitToRoom->getWeight())};
+        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_exitStatus, utils::richText(tr("A positive roomID of the room that this special exit leads to is expected here. "
```

Comment:
We don't really have to worry about positive/negative roomID's in Mudlet, it's not an issue folk struggle with, so we can shorten this -

```suggestion
        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_exitStatus, utils::richText(tr("roomID of the room that this special exit leads to is expected here. "
```

Reply:
Since it is an instruction giving guidance to the end-user it make sense to say what WILL work here, it is perfectly possible to enter a negative number (and indeed, slipping a `-` sign in might be the quickest way to invalidate an existing entry...); also, starting the sentence with "roomID" doesn't look right to me, as an English speaker I'd expect it to be capitalised - with the "A positive ..." before it though I avoided having to chose between "roomID" or "RoomID" 😜

Reply:
We want to be as concise as possible in these things, not verbose.

Reply:
Grammatically, if we are losing the "positive" then we still need the definite article preceding `roomID`:
```suggestion
        pSpecialExit->setToolTip(ExitsTreeWidget::colIndex_exitStatus, utils::richText(tr("The roomID of the room that this special exit leads to is expected here. "
```

Reply:
Done in https://github.com/Mudlet/Mudlet/pull/6102/commits/0fa9bfc6dc8c8a68a0954017baa7e4fea1195d8a .